.github/workflows/binaries-ea.yml

```diff
@@ -89,7 +89,7 @@ jobs:
             exit 0
           fi
 
-          if { [[ "$RUNNER_OS" == macOS ]] && ( [[ "${GITHUB_REF}" == refs/tags/* ]] || [[ "${{ inputs.notarization }}" == "true" ]] ); }; then
+          if { [[ ( [[ "${GITHUB_REF}" == refs/tags/* ]] || [[ "${{ inputs.notarization }}" == "true" ]] ); }; then
```

Comment:
why do we suddenly check for tags??

Reply:
We had this in the old run, too. --> https://github.com/JabRef/jabref/blob/1eb3493f9dfe19c42b5879eb755a830757c81cba/.github/workflows/deployment.yml#L226

Reply:
hm, I wonder if we still need this. I think we used this as part of the release process to use only tag builds but with the manual activation switch I think it is no longer needed

Reply:
I am in favor of keeping it: Fully automated release builds without the need of any manual trigger!